src/components/Footer/Creditos.tsx

```diff
@@ -195,6 +192,56 @@ const Creditos = () => {
               ))}
             </PopoverBody>
           )}
+
+          <PopoverHeader border="none" borderTop="1px solid" borderColor="electivas.200">
```

Comment:
```suggestion
    <PopoverHeader
        border="none"
        borderTop="1px solid"
        borderColor={"electivas.50"}
        mx={1}
      >
```

Que pensás de este estilado?  El color `electivas.200` no existe en la app!

![image](https://github.com/user-attachments/assets/7ab29966-bc53-48b5-8dfd-16bf0c142fc5)